src/axom/quest/interface/internal/QuestHelpers.cpp

```diff
@@ -13,8 +13,12 @@
   #include "axom/quest/readers/PSTLReader.hpp"
 #endif
 
-#if defined(AXOM_USE_MPI) && defined(AXOM_USE_C2C)
-  #include "axom/quest/readers/PC2CReader.hpp"
+#if defined(AXOM_USE_C2C)
+  #if defined(AXOM_USE_MPI)
+    #include "axom/quest/readers/PC2CReader.hpp"
```

Comment:
```suggestion
    #include "axom/quest/readers/C2CReader_MPI.hpp"
```
Would changing the file name make it clearer? 'PC2C..." seems too terse to me.

Reply:
Note: We also have an `STLReader` and a corresponding `PSTLReader`.

I think renaming the class with the `_MPI` suffix might obscure the intent. 
i.e. the "parallel" part is the important part, rather than "MPI"

Would you prefer something like `Parallel{C2C,STL}Reader`, or perhaps `Par{C2C,STL}Reader` ?

Reply:
Thoughts from others about (re)-naming these files and classes? 

I'd be ok with any of the following:
1. PSTLReader / PC2CReader (current, keep as is)
2. ParSTLReader / ParC2CReader
3. ParallelSTLReader / ParallelC2CReader
4. STLReader_MPI / C2CReader_MPI

Additional notes/factors to consider: 
* At least some users are using `PSTLReader` and would need to make changes
* The way the class works is that it reads the file on rank 0 and sends the data to the other ranks.

Reply:
If we don't have immediate consensus, perhaps it would be best to move this to a separate issue so we can have a more focused discussion (?)

Reply:
Thanks @agcapps -- I also vote for leaving it as is for now. 

I'll create an issue to continue the conversation.